docs/source/plugins/05_solver_config_hooks.rst

```diff
@@ -33,7 +33,7 @@ the hook listed below, the plugin can:
  - Add/update layers
 
 .. note::
-    For each new added ``field`` is considered a mass conservation equation and for each new added ``layer`` is considered
+    For each new added ``field`` is considered a mass conservation equation and for each new added ``layer`` are considered
     a momentum conservation and an energy conservation equations, depending on the energy model used at |alfasim|.
```

Comment:
This phrase is still ambiguous. A suggestion could be:

```suggestion
    For each new added ``field``, an extra mass conservation equation is considered. For each new added ``layer``, an extra momentum and energy conservation equations are considered, depending on the energy model used at |alfasim|.
    
```

Reply:
Done.